modules/xmpp/JingleSessionPC.js

```diff
@@ -1975,54 +1955,43 @@ export default class JingleSessionPC extends JingleSession {
     /**
      * Parse the information from the xml sourceRemoveElem and translate it
      *  into sdp lines
-     * @param {jquery xml element} sourceRemoveElem the source-remove
+     * @param {Element[]} sourceRemoveElem the source-remove
      *  element from jingle
-     * @param {SDP object} currentRemoteSdp the current remote
+     * @param {Object} currentRemoteSdp the current remote
      *  sdp (as of this new source-remove)
      * @returns {list} a list of SDP line strings that should
      *  be removed from the remote SDP
      */
     _parseSsrcInfoFromSourceRemove(sourceRemoveElem, currentRemoteSdp) {
         const removeSsrcInfo = [];
 
-        $(sourceRemoveElem).each((i1, content) => {
-            const name = $(content).attr('name');
+        $$_(sourceRemoveElem).forEach(content => {
```

Comment:
Why do we need to wrap it in `$$_`?

```suggestion
        sourceRemoveElem.forEach(content => {
```